### ISSUE TITLE
The class 'App\Entity\System\Settings' was not found in the chain configured namespaces App\Entity\Master

### ISSUE THREAD

**Issue by pedrocasado** [jbtronics/settings-bundle#16]
The class 'App\Entity\System\Settings' was not found in the chain configured namespaces App\Entity\Master

This error happens when we have 2 entity managers configured in doctrine.yaml

```
entity_managers:
    master:
        connection: master
        mappings:
            Master:
                is_bundle: false
                type: attribute
                dir: '%kernel.project_dir%/src/Entity/Master'
                prefix: 'App\Entity\Master'
                alias: Master

    default:
        connection: default
        mappings:
            App:
                is_bundle: false
                type: attribute
                dir: '%kernel.project_dir%/src/Entity/System'
                prefix: 'App\Entity\System'
                alias: App
```

I think this is because `ORMStorageAdapter` injects EntityManagerInterface and looks like it uses a default connection (the first one declared) which is not the correct one.

https://github.com/jbtronics/settings-bundle/blob/a3acdef1fdf6f1707e5fbd1fcef608870e4f39f6/src/Storage/ORMStorageAdapter.php#L50

I think `storageAdapterOptions` can have a `connection` option so we can specify which entity manager should be used. 

```
#[Settings(storageAdapter: ORMStorageAdapter::class, storageAdapterOptions: ['entity_class' => 'App\Entity\System\Settings', 'connection' => 'foo'])]
class AppSettings
{
}
```

In addition you could use `use Doctrine\Persistence\ManagerRegistry;` instead of `EntityManagerInterface` as it has the `getConnection` method.

Does it make sense?

**Comment by jbtronics** [comment]
Yes using ManagerRegistry sounds reasonable, and allowing to configure which connection to use, seems reasonable

**Comment by jbtronics** [comment]
This is fixed with version 2.3.0.

Normally, the entity manager to use should be detected automatically based on the entity class.

If you want to override this for some reason you can use the `entity_manager` option to specify the name of the entity manager to use
